qemu/tests/cfg/invalid_cpu_device_hotplug.cfg

```diff
@@ -0,0 +1,36 @@
+- invalid_cpu_device_hotplug:
+    only x86_64 ppc64 ppc64le
+    virt_test_type = qemu
+    type = invalid_cpu_device_hotplug
+    required_qemu = [2.6.0, )
+    # ovmf does not support hotpluggable vCPU yet
+    no ovmf
+    qemu_sandbox = on
+    vcpu_devices = vcpu1
+    monitor_type = qmp
+    variants:
+        - in_use_id:
+            execute_test = in_use_vcpu
+            ppc64, ppc64le:
+                error_desc = "core {0} already populated"
+            x86_64:
+                Windows:
+                    no WinXP, WinVista, Win7, Win8, Win10, Win2000, Win2003
+                error_desc = "CPU[{0}] with APIC ID"
```

Comment:
I prefer "CPU[{0}] with APIC ID {0} exists", it looks more like an error message.

Reply:
`APIC ID` is not always equal to `{0}`, so I need to circumvent it.

Reply:
How about using re patterns? Not sure if current short desc would match other errors unexpectedly.

Reply:
Sure, this is another way to check.